<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorClass</class>
 <widget class="QWidget" name="ItemEditorClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>项目编辑</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTableView" name="itemTable">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>添加</string>
          </property>
          <property name="icon">
           <iconset resource="../Cosmetic/Cosmetic.qrc">
            <normaloff>:/common/Resources/gtk-add.png</normaloff>:/common/Resources/gtk-add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>移除</string>
          </property>
          <property name="icon">
           <iconset resource="../Cosmetic/Cosmetic.qrc">
            <normaloff>:/common/Resources/gtk-remove.png</normaloff>:/common/Resources/gtk-remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submitButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>提交</string>
          </property>
          <property name="icon">
           <iconset resource="../Cosmetic/Cosmetic.qrc">
            <normaloff>:/common/Resources/ok.png</normaloff>:/common/Resources/ok.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Cosmetic/Cosmetic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>ItemEditorClass</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>ItemEditorClass</receiver>
   <slot>remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitButton</sender>
   <signal>clicked()</signal>
   <receiver>ItemEditorClass</receiver>
   <slot>remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>remove()</slot>
  <slot>submit()</slot>
 </slots>
</ui>
